config/puma.rb

```diff
@@ -42,3 +42,11 @@
 
 # Allow puma to be restarted by `rails restart` command.
 plugin :tmp_restart
+
+pidfile "/tmp/puma.pid"
```

Comment:
makes sense, but to ensure that you need to configure the state_path as well. i don't know where it will be stored by default, but i think it's not in `/tmp`